syft/frameworks/torch/mpc/falcon.py

```diff
@@ -0,0 +1,82 @@
+import torch
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+from syft.workers.base import BaseWorker
+
+from typing import List
+
+
+class Falcon(object):
+    @staticmethod
+    def _share_random_tensor(
+        field: int, workers: List[BaseWorker]
+    ) -> (ReplicatedSharingTensor, ReplicatedSharingTensor):
+        """
+        TODO: Need to move this in Primitive Store:
```

Comment:
this function is returning two different sharing which is not indicated from the name of the function and also makes it less general. why not return a single sharing and call the function twice?

Reply:
Hmm..in that case I think I can completely remove this function - doing that now